src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance.mdx

```diff
@@ -45,7 +45,7 @@ Don't have an account, but want a high level overview of your entire system? <In
 />
 
 <figcaption>
-  Know exactly where to focus your attention with New Relic Lookout. The brighter the color, the more severe the change, and the bigger the circle size, the greater the deviation in signal. Then dig deeper with correlations and abnormal history to see how it impacts your whole system—no configuration needed.
+  Know exactly where to focus your attention with New Relic Lookout. The brighter the circle color, and the bigger the circle size, the greater the deviation in signal. Then dig deeper with correlations and abnormal history to see how it impacts your whole system—no configuration needed.
```

Comment:
This seems important. I think it should be outside the fig caption, or if it's already outside the fig caption then you can prob delete it here